client/src/components/NavBar.tsx

```diff
@@ -42,7 +61,9 @@ export const NavBar: React.FC<NavBarProps> = ({}) => {
                     variant="link">Logout</Button>
                 <Tag backgroundColor={"transparent"} mr={2}>{data.me.username}</Tag>
                 <NextLink href={"/character"}>
-                    <Link alignContent='center' color={"white"} mr={2}><CharacterIcon/></Link>
+                    {/* <Link alignContent='center' color={"white"} mr={2}><CharacterIcon /></Link> */}
```

Comment:
What is this charactericon? Is it the old version of seticon? If so you can delete it